### PR TITLE
replace deprecations with new funcs

### PR DESCRIPTION
**Pull request by ake123**

### CODE REVIEW

File: R/plotDMN.R [hunk 1]
```diff
@@ -55,7 +55,7 @@ setMethod("plotDMNFit", signature = c(x = "SummarizedExperiment"),
             dmn <- metadata(x)[[name]]$dmm
         } else {
             .Deprecated(old="getDMN", new="cluster", 
-                    "Now runDMN and calculateDMN are deprecated. Use cluster with DMMParam parameter and full parameter set as true instead.")
+                    "Now runDMN and cluster are deprecated. Use cluster with DMMParam parameter and full parameter set as true instead.")
```
Comment (TuomasBorman) [inline R/plotDMN.R:58]: Here is error

Comment (TuomasBorman) [reply R/plotDMN.R:58]: Also add msg="Now runDMN...." --> the deprecation message is not printed correctly (I have noticed that in different functions)

**Comment by antagomir** [comment]
@ake123 could you finalize this one so we can close?

File: tests/testthat/test-2plotGraph.R [hunk 2]
```diff
@@ -50,7 +50,7 @@ test_that("plot graph", {
     expect_equal(miaViz:::.colnames_tbl_graph(actual,"edges"),
                  c("from","to","weight"))
     #
-    genus <- agglomerateByRank(GlobalPatterns,"Genus",na.rm=TRUE)
```
Comment (TuomasBorman) [inline tests/testthat/test-2plotGraph.R:53]: This is incorrect. mergeFeaturesByRank will be removed

**Comment by TuomasBorman** [comment]
This PR replaces agglomerateByRank with mergeFeaturesByRank, however, the mergeFeaturesByRank will be deprectated. (I have not updated Bioc devel of mia since we have still some modifications in GH that are under discussion)

Also, the cluster function will be deprecated so this PR has only old information. Sorry for that

**Pull request closed without merging**